### DISCUSSION TITLE
No development build only for T3 Turbo

### DISCUSSION THREAD

**Discussion by ghost** [t3-oss/create-t3-turbo#1197] [Q&A] [answered]
Hi guys, I have started a new project, but when pressing 'a' for expo, it fails with an error.

"No development build"

![image](https://github.com/user-attachments/assets/d569876e-2a20-48fd-abc0-9361a7c4e8df)

When I don't use T3 Turbo and just start a Expo project it works correctly.

If someone explain how do I get the development build working I would greatly appreciate it?

**Comment by edenrosales** [comment, marked as answer]
[This](https://docs.expo.dev/develop/development-builds/introduction/) provides some context to the issue. 
You need to build and install a development version of your application onto a test device if you are using development builds. To my understanding, this is the best practice, unless you intend to not use any native libraries not developed by the expo team themselves.

**Reply by ghost** [reply]
@edenrosales thanks for the feedback and the link, it put me on the right track, thanks!
